ggshield/cmd/sca/scan/ci.py

```diff
@@ -67,11 +69,15 @@ def scan_ci_cmd(
             config.user_config.verbose
         )
 
+        ci_mode = SupportedCI.from_ci_env()
+        scan_mode = f"{ScanMode.CI.value}/{ci_mode.value}"
         result = sca_scan_diff(
             ctx=ctx,
             directory=directory,
             previous_ref=previous_commit,
             current_ref=current_commit,
+            scan_mode=scan_mode,
+            ci_mode=ci_mode.name,
```

Comment:
`ci_mode` is already contained in `scan_mode`, this value is not necessary I think

Reply:
I know but it's what is done in secrets so I took it, I thought the variable `Ci-Mode` might be used in gim backend but I need to check that

Reply:
Ok then !